.pre-commit-config.yaml

```diff
@@ -64,6 +64,15 @@ repos:
     hooks:
       - id: yesqa
 
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.2
+    hooks:
+      - id: codespell
+        args: ["--write-changes"]
+        exclude: "extern|cextern"
+        additional_dependencies:
+          - tomli
```

Comment:
The codespell action with `--write-changes` should precede `black` because it can potentially change line length.

Reply:
A good point! I need to change my repos now 😆